src/Stripe.net/Enums/Billing.cs

```diff
@@ -4,6 +4,7 @@ namespace Stripe
     using Newtonsoft.Json;
     using Newtonsoft.Json.Converters;
 
+    // Deprecated. Please use CollectionMethod
```

Comment:
Ha, so obviously I didn't remember we had an enum for this field. We likely should do the same enum for CollectionMethod after all.

@ob-stripe Do you think we should have an enum? If we do, do you think we need one per resource, or `InvoiceCollectionMethod` should be used for all 3?
I really don't like `Billing` as being the unique name of that enum so I think the new one should at least be prefixed.

Reply:
I'd leave it as a string for now, it doesn't really make sense to use enums for just a couple of properties.